php_phongo.h

```diff
@@ -47,16 +47,21 @@ ZEND_BEGIN_MODULE_GLOBALS(mongodb)
 	HashTable clients;
 ZEND_END_MODULE_GLOBALS(mongodb)
 
+ZEND_EXTERN_MODULE_GLOBALS(mongodb)
+
 #if PHP_VERSION_ID >= 70000
-#   define MONGODB_G(v) ZEND_MODULE_GLOBALS_ACCESSOR(mongodb, v)
+# define MONGODB_G(v) ZEND_MODULE_GLOBALS_ACCESSOR(mongodb, v)
+# if defined(ZTS) && defined(COMPILE_DL_MONGODB)
+ZEND_TSRMLS_CACHE_EXTERN()
```

Comment:
You don't really need the "ZEND_EXTERN_MODULE_GLOBALS(mongodb)" though, as you've just declared the globals the few lines above. It is only there to put a forwards declaration in in places that normally can't "see" the globals.

I have never actually seen that PHP 7 needs a different thing here. I don't do that in Xdebug either...